### DISCUSSION TITLE
maven.pkg.github.com does not have a certificate<title>

### DISCUSSION THREAD

**Discussion by craigmunday** [community/community#119601] [Programming Help]
### Select Topic Area

General

### Body

When downloading a package from https://maven.pkg.github.com, I receive the error message.

"Your connection to this site isn't secure" and "This site does not have a certificate." 

I've tried this download from a number of networks.  Is there a TLS misconfiguration at this address?

**Comment by ebndev** [comment]
Hi @craigmunday,

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Thanks for posting in the GitHub Community!
